sonar-project.properties

```diff
@@ -1,3 +1,3 @@
-sonar.projectKey=etendosoftware_etendo_core_AYOKvQCAuJ79WHyLB97g
```

Comment:
Is needed to change sonar project key? This will cause sonar history reset.

Reply:
My bad, I didn't notice that change getting uploaded

Reply:
Done!